lib/matplotlib/axes/_axes.py

```diff
@@ -8205,7 +8205,7 @@ def matshow(self, Z, **kwargs):
     @_preprocess_data(replace_names=["dataset"])
     def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
                    showmeans=False, showextrema=True, showmedians=False,
-                   quantiles=None, points=100, bw_method=None):
+                   quantiles=None, points=100, bw_method=None, side='both'):
```

Comment:
I considered suggesting making the new argument kwonly here, but lean towards "don't bother" as I'd prefer to just get #27786 merged and then it will at least warn anyway.

Reply:
Ah but it can be kwonly. I have no preference.

Reply:
Let's not bother here. #27786 will take care.